src/python/py_llm_pipeline.cpp

```diff
@@ -130,7 +154,7 @@ void init_llm_pipeline(py::module_& m) {
             const py::kwargs& kwargs
         ) {
             ScopedVar env_manager(pyutils::ov_tokenizers_module_path());
-            ov::AnyMap properties = pyutils::kwargs_to_any_map(kwargs);
+            ov::AnyMap properties = llm_kwargs_to_any_map(kwargs);
```

Comment:
Why isn't `kwargs_to_any_map` modified instead?

Reply:
Wasn't sure that adapters are available for all pipelines. I can move it to common part.